packages/salesforcedx-vscode-core/package.json

```diff
@@ -29,7 +29,7 @@
     "@salesforce/salesforcedx-utils-vscode": "59.13.0",
     "@salesforce/schemas": "^1.6.1",
     "@salesforce/source-deploy-retrieve": "9.8.4",
-    "@salesforce/templates": "^59.0.4",
+    "@salesforce/templates": "^60.1.0",
```

Comment:
@WillieRuemmele this library change is going to pull in `@salesforce/core` v6. Are the template changes tied to 248 release of core, cuz I see the major bump?

@daphne-sfdc is working on the core v5-v6 change now, so this PR might have to wait until that work is complete before this PR can progress.

Reply:
they're not related to a 248 core version, but that was just the version of `templates` that was released when I merged. I believe `60` was already published, and then I published `60.1` - but it was automated

Reply:
@WillieRuemmele I published v60.0.1 less than an hour before you published - I realized that the Core v5 -> v6 change in the templates repo did not trigger a publish after the PR was committed, so I ran the manual release workflow.  Since your change in the templates repo is on top of that one, I believe your change would have triggered a v60 publish regardless of whether I did the manual release.